packages/web-app-files/src/components/SideBar/Actions/SpaceActions.vue

```diff
@@ -113,11 +113,21 @@ export default defineComponent({
 
 #oc-spaces-actions-sidebar {
   > li a,
-  > li a:hover {
+  > li a:hover { /* required? file actions are buttons not links. Looks like legacy */
```

Comment:
The actions can actually be links (e.g. the "Open folder"-action), so this is valid.

Reply:
ok. ty